cohortextractor/tpp_backend.py

```diff
@@ -1804,6 +1804,8 @@ def patients_with_these_codes_on_death_certificate(
             column_definition = "MIN(dod)"
         elif returning == "underlying_cause_of_death":
             column_definition = "MIN(icd10u)"
+        elif returning == "place_of_death":
+            column_definition = "MIN(Place_of_occurrence)"
```

Comment:
This arbitrarily uses `MIN` to handle possible duplicates with different `Place_of_occurrence` in the same way as `icd10u`.  This appears to be moot now - I checked the ONS_Deaths table and there are no duplicate Patient IDs. However, just in case that changes, we should probably continue to handle duplicates as we've done before